engine/src/main/java/org/terasology/rendering/dag/Node.java

```diff
@@ -15,7 +15,6 @@
  */
 package org.terasology.rendering.dag;
 
-
 public interface Node {
 
     void initialise();
```

Comment:
I just checked and this is... correct! I thought it was wrong and it should have been initialiZe, but it turns out initialise is British so, let's stick to the highest linguistic standard. =)